subsys/usb/device_next/class/usbd_uvc.c

```diff
@@ -0,0 +1,1145 @@
+				.bmControls = {0x00, 0x00, 0x00},                                  \
+			},                                                                         \
+		.if1 =                                                                             \
+			{                                                                          \
+				.bLength = sizeof(struct usb_if_descriptor),                       \
+				.bDescriptorType = USB_DESC_INTERFACE,                             \
+				.bInterfaceNumber = 1,                                             \
+				.bAlternateSetting = 0,                                            \
+				.bNumEndpoints = 1,                                                \
+				.bInterfaceClass = USB_BCC_VIDEO,                                  \
+				.bInterfaceSubClass = UVC_SC_VIDEOSTREAMING,                       \
+				.bInterfaceProtocol = 0,                                           \
+				.iInterface = 0,                                                   \
+			},                                                                         \
+		.if1_header =                                                                      \
+			{                                                                          \
+				.bLength = sizeof(struct uvc_stream_input_header_descriptor),      \
+				.bDescriptorType = UVC_CS_INTERFACE,                               \
+				.bDescriptorSubtype = UVC_VS_INPUT_HEADER,                         \
+				.bNumFormats = 0,                                                  \
+				.wTotalLength = sys_cpu_to_le16(                                   \
+					sizeof(struct uvc_stream_input_header_descriptor) +        \
+					sizeof(struct uvc_uncompressed_format_descriptor) +        \
+					sizeof(struct uvc_frame_descriptor)),                      \
+				.bEndpointAddress = 0x81,                                          \
+				.bmInfo = 0,                                                       \
+				.bTerminalLink = 2,                                                \
+				.bStillCaptureMethod = 0,                                          \
+				.bTriggerSupport = 0,                                              \
+				.bTriggerUsage = 0,                                                \
+				.bControlSize = 1,                                                 \
+				.bmaControls = {0x00},                                             \
+			},                                                                         \
+		.ep_fs =                                                                           \
+			{                                                                          \
+				.bLength = sizeof(struct usb_ep_descriptor),                       \
+				.bDescriptorType = USB_DESC_ENDPOINT,                              \
+				.bEndpointAddress = 0x81,                                          \
+				.bmAttributes = USB_EP_TYPE_BULK,                                  \
```

Comment:
I forgot to mention that only BULK is supported through this early version. I will need to spend more time investigating the `device_next` APIs for how ISO is implemented.

I cannot guarantee ETAs as I would do this feature on free time, but will add it to the roadmap: this will be needed at some point.